enterprise/api/v1/views.py

```diff
@@ -267,15 +375,10 @@ def list(self, request):
         Returns:
             (Response): DRF response object containing course catalogs.
         """
-        # fetch all course catalogs.
```

Comment:
@UmanShahzad I know what you mean -- ideally, comments should describe **why** something needs to happen; if the code itself is well-written, it shouldn't be necessary to add comments that describe **what** happens.

I hadn't looked too closely at the body of this method, but now that I have I agree that it makes sense to remove this particular comment :)